src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -30,7 +30,7 @@ interface NormalizerInterface
      * @param string $format  Format the normalization result will be encoded as
      * @param array  $context Context options for the normalizer
      *
-     * @return array|string|int|float|bool
+     * @return array|string|int|float|bool|\ArrayObject
```

Comment:
We should probably add a small sentence about when it returns an `array` vs an `ArrayObject` instance.